src/main/java/com/dococ/appointment/entity/Appointment.java

```diff
@@ -4,13 +4,17 @@
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
+import lombok.AllArgsConstructor;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
 import java.time.LocalDate;
 import java.time.LocalTime;
 
 @Entity
 @Data
+@AllArgsConstructor
```

Comment:
@Data includes AllArgsConstructor and NoArgsContructor by default so no need to include them separately